Tests/ValetIntegrationTests/ValetIntegrationTests.swift

```diff
@@ -284,6 +284,29 @@ class ValetIntegrationTests: XCTestCase
 
         XCTAssertNil(anotherFlavor.string(forKey: "cookie"))
     }
+
+    #if !os(macOS)
+    func test_objectForKey_canReadItemsWithout_kSecUseDataProtectionKeychain_when_kSecUseDataProtectionKeychain_isSetToTrueInKeychainQuery() {
+        let valet = Valet.valet(with: Identifier(nonEmpty: "DataProtectionTest")!, accessibility: .afterFirstUnlock)
+        var dataProtectionWriteQuery = valet.keychainQuery
+        #if swift(>=5.1)
+        preCatalinaWriteQuery[kSecUseDataProtectionKeychain as String] = nil
```

Comment:
`dataProtectionWriteQuery`?

Reply:
🤦‍♂️ copy/pate is fun
```suggestion
        dataProtectionWriteQuery[kSecUseDataProtectionKeychain as String] = nil
```